Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1309,6 +1315,9 @@
 /* No comment provided by engineer. */
 "NETWORK_STATUS_CONNECTING" = "Connecting";
 
+/* Error indicating that this device is no longer registered. */
+"NETWORK_STATUS_DEREGISTERED" = "No Longer Registered";
```

Comment:
In the common case, this happens because the user has registered on their other device. So maybe it would be good to clarify that it's the *device* that's no longer registered, not the user.

e.g. maybe "device deactivated" or  "device no longer active"

/cc @jlund

Reply:
Great point.